docs/learn/security.mdx

```diff
@@ -1,28 +1,25 @@
 ---
 description: Connect Security Properties
-title: 🔒 Security Properties
+title: Security
+icon: shield-keyhole
 ---
 
-<Frame>
-<img src='/img/slinky_math.jpeg' width="60%"/>
-</Frame>
+## Assumptions
 
-## ⛓️ Overall Assumptions
-
-✅ **Connect is as secure as the chain it is deployed on** - This is the meaning of a "restaked" oracle - Connect leverages all available
+**Connect is as secure as the chain it is deployed on** - This is the meaning of a "restaked" oracle - Connect leverages all available
 security that the chain can provide by using the existing validator set and tying updates to consensus via vote extensions.
 
-✅ **Prices require 2/3 participation from validators to be posted on chain** - We require a minimum of 2/3 of stake weight to contribute
+**Prices require 2/3 participation from validators to be posted on chain** - We require a minimum of 2/3 of stake weight to contribute
 to a price aggregation for that price to actually update chain state. This is the same stake weight required to commit blocks.
 
-✅ **Connect avoids any external dependencies** - Connect injects price data directly into the chain it's deployed on. This means there's no
+**Connect avoids any external dependencies** - Connect injects price data directly into the chain it's deployed on. This means there's no
 additional chain, token, relayer, or other outside actor dependency. The same operators that run the chain run the oracle.
 
-✅ **Connect price updates are atomic with liveness** - Since Connect data is signed atomically with `PreCommit` votes via Vote Extensions, chains can
+**Connect price updates are atomic with liveness** - Since Connect data is signed atomically with `PreCommit` votes via Vote Extensions, chains can
 configure their application (or parts of their application) to depend on per-block price updates. This makes almost all forms of oracle attacks impossible,
```

Comment:
Note for the future: it might  be cool to include one of those grids that compares the feature sets and requirements of various oracles like Connect, Pyth, Chainlink, etc

Reply:
opened https://linear.app/skip/issue/BLO-1567/product-comparison-table-for-docs